docs/customization/repository.rst

```diff
@@ -0,0 +1,102 @@
+Customizing Repositories
+========================
+
+.. warning::
+    In **Sylius** we are using both default Doctrine repositories and the custom ones.
+    Often you will be needing to add your very own methods to them. You need to check before which repository is your resource using.
+
+Why would you customize a Repository?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Different sets of different resources can be obtained in various scenarios in your application.
+You may need for instance:
+
+    * finding Orders by a Customer and a chosen Product
+    * finding Products by a Taxon
+    * finding Comments by a Customer
+
+How to customize a Repository?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Let's assume that you would want to find products that are the bestsellers in your shop.
+
+1. Create your own repository class under the ``AppBundle\Repository`` namespace.
```

Comment:
According to what we've had previously in the docs, nope -> http://docs.sylius.org/en/latest/bundles/general/overriding_repositories.html